packages/fxa-auth-server/config/index.ts

```diff
@@ -2077,7 +2077,7 @@ const convictConf = convict({
   },
   cms: {
     enabled: {
-      default: false,
+      default: true,
```

Comment:
Are we ready for this?  I think most devs don't have Strapi configured and don't need it.